tslib.js

```diff
@@ -222,11 +222,23 @@ var __importDefault;
         return cooked;
     };
 
+    var __getImportStarCache = function () {
+        if (typeof WeakMap !== "function") return null;
+
+        var cache = new WeakMap();
+        __getImportStarCache = function () { return cache; }
+        return cache;
+    };
+
     __importStar = function (mod) {
         if (mod && mod.__esModule) return mod;
+        if (mod === null || (typeof mod !== "object" && typeof mod !== "function")) return { "default": mod }

```

Comment:
As with the other comments, moving this line up one can simplify the previous line, and you're missing a semicolon at the end of the statement.